### PR TITLE
Add scripts for building Debian packages

### PR DESCRIPTION
**Pull request by marcinslusarz**
This change is [Reviewable](https://reviewable.io/reviews/gbuella/syscall_intercept/1)

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+Everything in this source tree is covered by the previous license
+with the following exceptions:
+
+* utils/cstyle (used only during development) licensed under CDDL.
```
Comment (GBuella) [inline debian/copyright]: The cstyle script is only distributed as part of the source, so it should not be mentioned in the package, right?

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Source: syscall-intercept
+Maintainer: Marcin Ślusarz <marcin.slusarz@intel.com>
+Section: misc
+Priority: optional
+Standards-version: 3.9.8
+Build-Depends: debhelper (>= 9)
+
+Package: libsyscall-intercept0
+Architecture: any
```
Comment (GBuella) [inline debian/control]: Shouldn't we specify x86_64 here?
I don't debian control files, but I suspect that would make sense for this package.

Comment (GBuella) [inline debian/control]: "I don't debian" -> "I don't know debian"

File: debian/copyright (hunk 1 above)
Comment (marcinslusarz) [inline debian/copyright]: Fixed

File: debian/control (hunk 2 above)
Comment (marcinslusarz) [inline debian/control]: Fixed.

**Pull request merged**